src/components.jl

```diff
@@ -774,16 +749,16 @@ end
 
 @deprecate curve_points coords
 
-coords(curve::BezierCurve, n::Integer = 30; range = [0,1]) = map(curve, Base.range(first(range), stop=last(range), length=n))
+coords(curve::BezierCurve, n::Integer=30; range=[0, 1]) = map(
+  curve, Base.range(first(range), stop=last(range), length=n)
+)
 
 # build a BezierCurve which leaves point p vertically upwards and arrives point q vertically upwards.
-# may create a loop if necessary.  Assumes the view is [0,1]
-function directed_curve(args...; kw...)
-    error("directed_curve has been moved to PlotRecipes")
-end
+# may create a loop if necessary.  Assumes the view is [0, 1]
+directed_curve(args...; kw...) = error("directed_curve has been moved to PlotRecipes")
```

Comment:
```suggestion
```
It might make sense to remove this function here if it isn't exported. At leas `PlotRecipes` needs to be changed to `GraphRecipes`